docs/source/conf.py

```diff
@@ -84,7 +84,7 @@
 
 
 if (hasattr(owslib, '__version__') and
-        LooseVersion(owslib.__version__) >= '0.19.2'):
+        parse_version(owslib.__version__) >= parse_version('0.19.2')):
```

Comment:
This is slightly annoying that we have to wrap everything ourselves now, rather than "parse_version" being able to cast the string to `Version` internally during the comparison...